integration_tests/e2e/updateInduction.cy.ts

```diff
@@ -0,0 +1,69 @@
+import HopingToGetWorkPage from '../pages/hopingToGetWork'
+import ReasonToNotGetWork from '../pages/reasonToNotGetWork'
+import QualificationsPage from '../pages/qualifications'
+import AdditionalTrainingPage from '../pages/additionalTraining'
+import InPrisonWorkPage from '../pages/inPrisonWork'
+import InPrisonEducationPage from '../pages/inPrisonEducation'
+import Page from '../pages/page'
+import CheckYourAnswersPage from '../pages/checkYourAnswers'
+import PlpWorkAndInterestsPage from '../pages/plpWorkAndInterestsPage'
```

Comment:
This is a new test (that was missing) that does a full end to end update of an existing Induction. The significance of this test is that it changes the journey (from long to short), which means it goes via the Check Your Answers page, which in turn loads the Induction from the API when the page is submitted (ie. the bug we are fixing here)

This is the test that is testing the implementation below 👇